data/campaigns/Eastern_Invasion/scenarios/08_Xenophobia.cfg

```diff
@@ -625,9 +627,9 @@
         {REMOVE_LABEL $unit.x $unit.y}
     [/event]
 #enddef
-    {GOLD_PICKUP 11 4  items/gold-coins-small.png 10 dwarves}
-    {GOLD_PICKUP 37 26 items/gold-coins-small.png 20 serpents}
-    {GOLD_PICKUP 38 10 items/gold-coins-small.png 20 orcs}
+    {GOLD_PICKUP 11 4  items/gold-coins-small.png 10 (_"10 gold") (_"These dwarves have a pile of 10 gold pieces!")}
+    {GOLD_PICKUP 37 26 items/gold-coins-small.png 20 (_"20 gold") (_"These nagas have a pile of 20 gold pieces!")}
```

Comment:
I swapped "serpents" to "nagas", because in this scenario the human side is being respectful of the other races, in contrast to the dwarves', nagas' and orcs' lines.

Reply:
Aren't we breaking string freeze by adding more strings here? And below?

Reply:
I think it does technically break string freeze, but these new strings replace old strings that could not be translated, so I believe it's worth it.

Reply:
These were put into strings via macro, they're specifically requested for fixing #8225.